src/SFA.DAS.AssessorService.Settings/ConfigurationService.cs

```diff
@@ -8,7 +8,26 @@ namespace SFA.DAS.AssessorService.Settings
 {
     public static class ConfigurationService
     {
-        public static async Task<IWebConfiguration> GetConfig(string environment, string storageConnectionString, string version, string serviceName)
+        public static async Task<IApiConfiguration> GetConfigApi(string environment, string storageConnectionString, string version, string serviceName)
+        {
+            var config = await GetConfig<ApiConfiguration>(environment, storageConnectionString, version, serviceName);
+            config.Environment = environment;
+            return config;
+        }
+
+        public static async Task<IExternalApiConfiguration> GetConfigExternalApi(string environment, string storageConnectionString, string version, string serviceName)
+        {
+            var config = await GetConfig<ExternalApiConfiguration>(environment, storageConnectionString, version, serviceName);
+            return config;
+        }
+
+        public static async Task<IWebConfiguration> GetConfigWeb(string environment, string storageConnectionString, string version, string serviceName)
+        {
+            var config = await GetConfig <WebConfiguration>(environment, storageConnectionString, version, serviceName);
+            return config;
+        }
+
+        private static async Task<T> GetConfig<T>(string environment, string storageConnectionString, string version, string serviceName)
```

Comment:
We should have some tests for this new code that returns differernt types of configs